packages/functional-tests/tests/subscription-tests/coupons-ui-tests/resubscription.spec.ts

```diff
@@ -125,4 +125,43 @@ test.describe('resubscription test', () => {
     //Verify that the card info is updated
     expect(await subscriptionManagement.getCardInfo()).toContain('4444');
   });
+
+  test('update mode of payment for paypal', async ({
+    page,
+    pages: { relier, subscribe, login, settings, subscriptionManagement },
+  }) => {
+    await relier.goto();
+    await relier.clickSubscribe();
+
+    // 'auto10pforever' is a 10% forever discount coupon for a 6mo plan
+    await subscribe.addCouponCode('auto10pforever');
+
+    //Subscribe successfully
+    await subscribe.setPayPalInfo();
```

Comment:
I was going to say this `PayPal` is inconsistent with other `Paypal` usages, but then #14692 and it's a nit not worth refactoring any tests for.